src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts

```diff
@@ -55,7 +55,7 @@ export class TaskListFilterComponent implements OnInit, OnDestroy {
   public toggleFilter = false;
   public errorSubscription: Subscription;
   private subscription: Subscription;
-  private selectedLocationsSubscription: Subscription;
+  private sub: Subscription;
```

Comment:
Query - Why has this been changed back? Need more descriptive name

Reply:
Sure. Will change it back